solutions/instances/version.tf

```diff
@@ -0,0 +1,14 @@
+terraform {
+  required_version = ">= 1.3.0, <1.7.0"
+  required_providers {
+    # Pin to the lowest provider version of the range defined in the main module to ensure lowest version still works
```

Comment:
For a DA, we will always lock to the latest version, and we will add a renovate rule to ensure it stays updated. Please update this comment to:

`# Lock DA into an exact provider version - renovate automation will keep it updated`

And please create a renovate rule in the renovate.json file to keep them updated. Use the exact same code that is [here](https://github.com/terraform-ibm-modules/terraform-ibm-scc-da/blob/main/renovate.json)